config/config.go

```diff
@@ -1,11 +1,19 @@
 package config
 
 import (
+	"github.com/ONSdigital/dp-feedback-api/sdk"
 	"time"
 
 	"github.com/kelseyhightower/envconfig"
 )
 
+// Config for feedback sdk
+type FeedbackConfig struct {
+	BindAddr         string
+	ServiceAuthToken string
+	Client           *sdk.Client
```

Comment:
I think this should be removed, as this isn't a configuration, instead it's the client (should be created in `service` package when the service starts,using the other values, and provided to the handler so it can contact feedback api)

Reply:
Removed.